src/docs/cmdstan-guide/json-format.tex

```diff
@@ -0,0 +1,111 @@
+\chapter{JSON Format}\label{json.chapter}
+
+\noindent 
+\CmdStan can use JSON format for input data for both
+model data and parameters. Model data is read in by the model
+constructor. Model parameters are used to initialize the sampler and
+optimizer.
+
+\section{Stan Data Types in JSON Notation}
+
+A Stan input file in  JSON notation consists of  JSON object which contains zero
+or more name-value pairs.  This structure corresponds to a Python data
+dictionary object.  The following is an example of JSON data for the
+simple Bernoulli example model:
+
+%
```

Comment:
[question, possible action required]
What are the `%` for?  If this is just holdover from LaTeX spacing, please remove.

Reply:
done